mkosi/__init__.py

```diff
@@ -1697,6 +1697,8 @@ def fixup_vmlinuz_location(context: Context) -> None:
         for d in context.root.glob(f"boot/{type}-*"):
             kver = d.name.removeprefix(f"{type}-")
             vmlinuz = context.root / "usr/lib/modules" / kver / f"{type}"
```

Comment:
While we're at it, can we also shorten this?
```suggestion
            vmlinuz = context.root / "usr/lib/modules" / kver / type
```